src/Raven.Server/Documents/Handlers/StatsHandler.cs

```diff
@@ -56,5 +57,12 @@ public async Task BytesMetrics()
             using (var processor = new StatsHandlerProcessorForGetMetricsBytes(this))
                 await processor.ExecuteAsync();
         }
+
+        [RavenAction("/databases/*/validate-unused-ids", "GET", AuthorizationStatus.ValidUser, EndpointType.Read)]
+        public async Task ValidateUnusedIds()
+        {
+            using (var processor = new ValidateUnusedIdsHandlerProcessorForGet(this))
```

Comment:
check the name convention here.. should be `StatsHandlerProcessorForValidateUnusedIds`

Reply:
Done